alacritty_terminal/src/tty/unix.rs

```diff
@@ -232,7 +234,11 @@ pub fn new(config: &PtyConfig, window_size: WindowSize, window_id: Option<usize>
         },
         Err(err) => Err(Error::new(
             ErrorKind::NotFound,
-            format!("Failed to spawn command '{}': {}", shell.program(), err),
+            format!(
+                "Failed to spawn command '{}': {}",
+                builder.get_program().to_string_lossy(),
```

Comment:
This will just print `login` on macOS, right?

Reply:
Right, that's why I asked earlier if we should print the whole command, but you replied that we should not.

Reply:
Pushed a version that just prints the whole command, and re-uses the triggering error's `kind`, rather than assuming `NotFound`.

Reply:
No printing just `login` is fine, I was just trying to confirm its behavior.

Pushing the whole command will just be confusing to users, considering there's a bunch of exec and argument concatenation going on, it might even make some macOS users think there's something wrong with their Alacritty.

Reply:
Sure, will leave it.